source/ui/GuiApplication.ooc

```diff
@@ -18,7 +23,13 @@ GuiApplication: class extends Application {
 			this _eventLoop free()
 		super()
 	}
+	createWindow: virtual func (windowSize: IntVector2D, windowName: String) {
```

Comment:
I think so, anyone who has an application object can now create the window anytime he wants, not only in constructor.